cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorTest.java

```diff
@@ -146,6 +147,8 @@ public void testDelayedCheck() throws InterruptedException {
     Properties props = KafkaCruiseControlUnitTestUtils.getKafkaCruiseControlProperties();
     KafkaCruiseControlConfig kafkaCruiseControlConfig = new KafkaCruiseControlConfig(props);
     EasyMock.expect(mockKafkaCruiseControl.config()).andReturn(kafkaCruiseControlConfig).times(1, 2);
+    EasyMock.expect(mockKafkaCruiseControl.getLoadMonitorTaskRunnerState())
+            .andReturn(LoadMonitorTaskRunner.LoadMonitorTaskRunnerState.RUNNING).anyTimes();
```

Comment:
Is this really expected to be called `anyTimes()`? Would it be possible to be more conservative? (applies to the other places).

Reply:
changed to `times()`